### PR TITLE
docs: add information on how to run universal in conjunction with service worker

### PR DESCRIPTION
**Pull request by jessicajaniuk**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/universal.md [hunk 1]
```diff
@@ -246,6 +246,11 @@ Similarly, without mouse or keyboard events, a server-side application can't rel
 The application must determine what to render based solely on the incoming client request.
 This is a good argument for making the application [routable](guide/router).
 
+<a id="service-worker"></a>
+### Universal and the Angular Service Worker
+
+If you are using Universal in conjunction with the Angular service worker, the behavior is different than the normal server side rendering behavior. The initial server request will be rendered on the server as expected. However, after that initial request, subsequent requests are handled by the service worker. For subsequent requests, the `index.html` file is served statically and bypasses server side rendering. This is an optimization to have a faster response time with improved Core Web Vitals performance.
```
Comment (jessicajaniuk) [inline aio/content/guide/universal.md:252]: I'm thinking that last line is probably unnecessary.

File: aio/content/guide/service-worker-getting-started.md [hunk 2]
```diff
@@ -40,6 +40,15 @@ ng build
 
 The CLI project is now set up to use the Angular service worker.
 
+## Service Worker with NG Universal Server Side Rendering
```
Comment (alan-agius4) [inline aio/content/guide/service-worker-getting-started.md:43]: Let not add this section and fix the underlying problem instead. 

https://github.com/angular/universal/pull/3081

Comment (jessicajaniuk) [reply aio/content/guide/service-worker-getting-started.md:43]: Well, that was fast! Thanks, @alan-agius4. I'll update the PR.

**Review by alan-agius4 — APPROVED** [review]
LGTM

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 1f7f7598d57bd6b1e814cbf575883a3b29811ca1.

**Pull request closed without merging**